remoter-server/src/numGenesVsNumUmis.r

```diff
@@ -51,29 +34,40 @@ source('utils.r')
 #'                                - p.level: which refers to  confidence level for deviation from the main trend
 #' @export return a list with the filtered seurat object by numGenesVsNumUmis, the config and the plot values
 
-task <- function(scdata, config, task_name, sample_id){
+task <- function(seurat_obj, config, task_name, sample_id){
+    print(paste("Running",task_name,"config: ",sep=" "))
+    print(config)
+    #The format of the sample_id is
+    # sample-WT1
+    # we need to get only the last part, in order to grep the object.
+    tmp_sample <- sub("sample-","",sample_id)
     # Check wheter the filter is set to true or false
     if (!as.logical(toupper(config$enabled)))
-        return(scdata)
+        return(seurat_obj)
```

Comment:
Can this work? Shouldn't the return value be a list with the seurat obj in $data?
The way it is handled in cellSizeDist filter seems the way to go

Reply:
Yes, I actually wanted to discuss this. I left it like this because this was the way it was on data ingest